fetcher/secrets.py

```diff
@@ -176,10 +176,10 @@ def __init__(self, *args):
                     raise _RefreshOauth2TokenError(
                         I18nMessage.trans(
                             "py.fetcher.secrets._refresh_oauth2_token.server_error.general",
-                            default="{service_id} responded with HTTP {status_number} {error}: {description}",
+                            default="{service_id} responded with HTTP {status_code} {error}: {description}",
```

Comment:
`{reason}`, not `{error}`, for consistency. Sorry I didn't notice this one sooner.